cluster.go

```diff
@@ -149,14 +149,35 @@ type ClusterConfig struct {
 	// If not provided, a default dialer configured with ConnectTimeout will be used.
 	Dialer Dialer
 
+	// Scylla-specific cluster options
+	Scylla ScyllaClusterOpts
+
 	// internal config for testing
 	disableControlConn bool
 }
 
+// Scylla-specific cluster options
+type ScyllaClusterOpts struct {
+	// EnableSourcePortBasedLoadBalancing controls if the driver should make use
+	// of the native_shard_aware_transport_port and native_shard_aware_transport_port_ssl
+	// ports (if they are properly configured on the node).
```

Comment:
s/the node/a node or just nodes
```suggestion
	// ports (if they are properly configured on nodes).
```